src/parser/type_checking.c

```diff
@@ -10172,7 +10172,7 @@ pt_eval_expr_type (PARSER_CONTEXT * parser, PT_NODE * node)
 	  common_type = (arg1_type == PT_TYPE_MAYBE && arg2_type != PT_TYPE_NULL) ? arg2_type : arg1_type;
```

Comment:
This issue is related to CUBRIDSUS-9134.
https://github.com/CUBRID/cubrid/commit/f892ae1212f3da880371289356bf27371334ffa5

I think PT_TYPE_MAYBE should be kept if arg1 or arg2 are PT_TYPE_NULL.
I recommend the following changes.
'''
if (common_type == PT_TYPE_MAYBE && arg1_type != PT_TYPE_NULL && arg2_type != PT_TYPE_NULL)
{
common_type = (arg1_type == PT_TYPE_MAYBE) ? arg2_type : arg1_type;
}
'''

Reply:
Ok, I modified.